.github/workflows/run-k8s-tests-on-aks.yaml

```diff
@@ -52,6 +57,10 @@ jobs:
       GH_PR_NUMBER: ${{ inputs.pr-number }}
       KATA_HOST_OS: ${{ matrix.host_os }}
       KATA_HYPERVISOR: ${{ matrix.vmm }}
+      # Set to install the KBS for attestation tests
+      KBS: ${{ matrix.kbs }}
+      # Set the KBS ingress handler (empty string disables handling)
+      KBS_INGRESS: ${{ matrix.kbs == 'true' && 'aks' || '' }}
```

Comment:
```suggestion
      # Set the KBS ingress handler to "aks" if we install the KBS, otherwise pass an empty string to disable it
      KBS_INGRESS: ${{ matrix.kbs == 'true' && 'aks' || '' }}
```